staging/src/k8s.io/client-go/util/workqueue/delaying_queue.go

```diff
@@ -36,8 +36,8 @@ type DelayingInterface interface {
 // NewDelayingQueue constructs a new workqueue with delayed queuing ability.
 // NewDelayingQueue does not emit metrics. For use with a MetricsProvider, please use
 // NewNamedDelayingQueue instead.
-func NewDelayingQueue() DelayingInterface {
-	return NewDelayingQueueWithCustomClock(clock.RealClock{}, "")
+func NewDelayingQueue(opts ...QueueOption) DelayingInterface {
```

Comment:
As stated in the comment, delaying queues initialized with this method don't emit any metrics, so we shouldn't need a MetricsProvider here.

Reply:
Ah, missed that comment, thanks. This is because the name is empty, and that implicitly means no metrics?

Reply:
If the name is empty you can't identify which queue the metrics are about, so I guess it would make sense not to have metrics in that case since they would essentially be unusable.